requirements/test.txt

```diff
@@ -12,3 +12,4 @@ black==19.10b0
 pre-commit>=1.0
 
 cloudpickle>=1.2
+nltk
```

Comment:
do we have a version limitation?

Reply:
okay, will limit with current latest version.

Reply:
it is fine for our CI, but for other users...
you can just send a commit with 1year / 2year old nltk version and see if tests are still passing :]

Reply:
Sorry about that as I am always used to latest version 😁

I have locked with `nltk==3.3` as tests failed on `3.2`.

Reply:
I agree that it is better but some users may freeze their version because of some functional requirements... :]
so let's make it `nltk>=3.3`

Reply:
Done it.